src/PTFAnalysis.cpp

```diff
@@ -719,7 +811,7 @@ PTFAnalysis::PTFAnalysis( TFile* outfile, Wrapper & wrapper, double errorbar, PT
 
 //       Do simple charge sum calculation
         if( pmt.pmt == 0 ) {
-            ChargeSum(0.9931); //original PTF function call here
+            ChargeSumPTF(8135.4); //original PTF function call here
```

Comment:
This should come after `FitWaveform` and use the resulting `Offset` as the pedestal (different for each waveform), instead of this constant value over all waveforms.

(Is this why your charge distribution noise peak is not centered at 0?)